CHANGELOG.md

```diff
@@ -551,7 +551,7 @@ Lots of cleanup and docs additions made it into this release. Parsing is simpler
 ## Version 0.3: Plumbum compatibility
 
 * Added `->requires`, `->excludes`, and `->envname` from [Plumbum](http://plumbum.readthedocs.io/en/latest/)
-* Supports `->mandatory` from Plubmum
+* Supports `->mandatory` from Plumbum
```

Comment:
Impressed it caught this one

Reply:
The tooling just reports tokens not in the dictionary, since neither are in the dictionary, I'm presented w/ two fairly close tokens in a long list. I just check the context for both and pick one as the more likely real token. As a human, I do reasonably well at picking, but I do fail periodically. Also, since I tend to do this late at night, I may skip over some set of tokens deciding "they're probably real".